libsolidity/interface/CompilerStack.h

```diff
@@ -289,7 +290,8 @@ class CompilerStack: boost::noncopyable
 	std::vector<Source const*> m_sourceOrder;
 	std::map<std::string const, Contract> m_contracts;
 	std::string m_formalTranslation;
-	ErrorList m_errors;
+	ErrorList m_errorList;
```

Comment:
That's actually the original ErrorList object that all the ErrorReporters hold on to by reference.